src/index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 7 Retina 5K (MNED2UA/A)"
+        class="card__image"
+      >
+      <h1 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </h1>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__block-stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__block-text">Reviews: 5</p>
+      </div>
+      <div class="card__block-price">
```

Comment:
Add an empty line between the blocks for better readability
```suggestion
      </div>
      
      <div class="card__block-price">
```

Reply:
fixed